tests/end-to-end/cli/filter/filter-dataprovider-by-phpstorm-regexp.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
```

Comment:
we need another test with a numbered dataprovider.

and I think we should land the tests on PHPUnit 11+ to make sure all PHPUnit versions are on the same page

Reply:
Maybe even for PHPUnit >= 10.